pontoon/base/models.py

```diff
@@ -1225,6 +1231,10 @@ def for_project_locale(self, project, locale, paths=None, filter_type=None,
             elif filter_type in Translation.authors(locale, project, paths).values_list('email', flat=True):
                 entities = self.objects.authored_by(locale, filter_type)
 
+            elif re.match('^[0-9]{12}-[0-9]{12}$', filter_type):
+                start, end = utils.parse_time_interval(filter_type)
```

Comment:
I think we could return an error/ignore this filter if value is malformed somehow e.g. start > end